src/front/macOS/kDrive/Controllers/PreferencesWindow/SplitView/PreferencesSidebarViewController.swift

```diff
@@ -0,0 +1,116 @@
+/*
+ Infomaniak kDrive - Desktop
+ Copyright (C) 2023-2025 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import Cocoa
+import kDriveCoreUI
+
+extension SidebarItem {
+    static let general = SidebarItem(icon: NSImage(resource: .settings), title: KDriveLocalizable.sidebarItemGeneral)
+    static let accounts = SidebarItem(icon: NSImage(resource: .persons), title: KDriveLocalizable.sidebarItemAccounts)
+    static let advanced = SidebarItem(icon: NSImage(resource: .hammerWrench), title: KDriveLocalizable.sidebarItemAdvanced)
+}
+
+class PreferencesSidebarViewController: NSViewController {
+    static let navigationCellIdentifier = NSUserInterfaceItemIdentifier("NavigationSidebarCell")
```

Comment:
I think I prefer the following, as the compilation will fail on a partial rename. If you agree you can apply this pattern on each of your `NSUserInterfaceItemIdentifier`

```suggestion
    static let navigationCellIdentifier = NSUserInterfaceItemIdentifier(String(describing: NavigationSidebarCell.self))
```